L/LLVMLibUnwind/LLVMLibUnwind@14.0.6/bundled/patches/0003-stdc-format-macros.patch

```diff
@@ -0,0 +1,10 @@
+--- a/libunwind/DwarfParser.hpp
++++ b/libunwind/DwarfParser.hpp
+@@ -12,6 +12,7 @@
+ #ifndef __DWARF_PARSER_HPP__
+ #define __DWARF_PARSER_HPP__
+ 
++#define __STDC_FORMAT_MACROS
```

Comment:
They use it elsewhere, but not in libunwind: https://github.com/llvm/llvm-project/blob/5d87f60f894764bd435086e311605884638d2384/libcxx/include/inttypes.h#L244-L249